deploy.ci.acceptance.dynamic-store-off.yml

```diff
@@ -58,7 +58,7 @@ groups:
     EU:
         region: EU
         applications:
-            yves:
+            yves_eu:
```

Comment:
why do we need to rename here? especially since this is dms off and there is no eu (there should be de)